test/unit/model/config-test.js

```diff
@@ -0,0 +1,26 @@
+import Model from 'controller/model';
```

Comment:
This file should be called "model-test". We already have "api-config" and "api-get-gonfig" which test the config module, and the `getConfig()` method which returns a clone of the model attributes after `api.setup()`.